src/shared/components/complex/gokb-reviews-title-card/gokb-reviews-title-card.vue

```diff
@@ -0,0 +1,293 @@
+          { text: this.$i18n.tc('component.identifier.label', 1), align: 'start', value: 'value', sortable: false }
+        ]
+      },
+      idsEditable () {
+        return this.isCardSelected || (this.role != "candidateComponent" && this.isOtherCardSelected)
+      },
+      historyHeaders () {
+        return [
+          { text: this.$i18n.tc('component.title.history.date.label'), align: 'start', value: 'date', sortable: false },
+          { text: this.$i18n.tc('component.title.history.from', 1), align: 'start', value: 'from', sortable: false },
+          { text: this.$i18n.tc('component.title.history.to', 1), align: 'start', value: 'to', sortable: false }
+        ]
+      },
+      historyVisible () {
+        return [...this.originalRecord?.history]
+          .map(item => ({
+            namespace: item.namespace.value,
+            value: item.value
+          }))
+      },
+      totalNumberOfIds () {
+        return !!(this.originalRecord?._embedded?.ids) ? this.originalRecord._embedded.ids.length : 0
+      },
+      historyEditable () {
+        return false
+      },
+      getHistory () {
+        let result = []
+        if (!!this.originalRecord?.history) {
+          this.originalRecord.history.forEach(h => {
+            h.from.forEach((from, i) => {
+              result.push({
+                id  : h.id,
+                date: h.date,
+                from: h.from[i].name,
+                to  : h.to[i]?.name
+              })
+            })
+          })
+        }
+        // TODO: add Coverage
+        return result
+      },
+      roleColor () {
+        if (this.role == "reviewedComponent") {
+          return "#ffc1c1"
+        }
+        if (this.role == "candidateComponent" && this.isCardSelected) {
+          return "#c1ffc1"
+        }
+      },
+      isOtherCardSelected () {
+        return !!this.id && !!this.selCard && this.id != this.selCard
+      },
+      isItemDeletable () {
+        if (this.isReviewedCard) {
+          return undefined
+        }
+        else return true
+      },
+      isCandidate () {
+        return this.role != 'reviewedComponent'
+      }
+    },
+    watch: {
+      selCard () {
+        this.isCardSelected = !!this.id && this.id == this.selCard
+      },
+      selectedCard () {
+        this.selCard = this.selectedCard
+      },
+      selectedCardIds () {
+        this.selCardIds = this.selectedCardIds
+      },
+      selectedIdItems () {
+        this.$emit('reviewedCardSelectedIds', this.selectedIdItems)
+      }
+    },
+    created () {
+      this.isReviewedCard = this.role == "reviewedComponent"
+    },
+    async mounted () {
+      this.fetchTitle(this.id)
+    },
+    beforeUpdate () {
+      this.idsVisible = this.visibleIdentifiers()
+    },
+    methods: {
+      async fetchTitle (tid) {
+        this.finishedLoading = false
+        const response = await this.catchError({
+          promise: titleServices.getTitle(tid, this.cancelToken.token),
```

Comment:
This might also be a TIPP in other review types, or you would need to conditionally use an `gokb-reviews-tipp-card` instead in the `gokb-reviews-title-section`

Reply:
... this Is part of the considerations for other RR cases.